src/engraving/iengravingconfiguration.h

```diff
@@ -93,7 +93,11 @@ class IEngravingConfiguration : MODULE_EXPORT_INTERFACE
         bool colorSegmentShapes = false;
         bool showSkylines = false;
         bool showSystemBoundingRects = false;
+#ifndef NDEBUG
         bool showCorruptedMeasures = true;
+#else
+        bool showCorruptedMeasures = false;
+#endif
```

Comment:
Not really sure about this, having it set to true seems a sane default for debug and non-debug builds, esp. as that setting doesn't seen to get saved accross sessions, so needs to get enabled time and time again (in non-debug builds)
Another or an additional option might be to have a a checkbox in the 'Corrupt score' dialog, to enable this setting right when it is needed, or even enable it there without asking the user.

Reply:
As a "stretch goal", we could do the following:
- Indeed, make this option true by default
- Add a command to (re-)check for corruptions (current workaround is to wait until Autosave runs, because that also checks for corruptions, but does not warn about them in the UI, and it is questionable whether Autosave is supposed to run the corruption check at all)
- For optimisation, set a member variable in the `Score` whether it contains any corrupted measures; if that's false, no need to iterate over all measures to find corrupted ones

Reply:
The true by default is dead easy, I'll do it right way
Not sure how to go about adding such a command (will look into it), but indeed autosave shouldn't run that check or report them right away.
The optimization won't help much, as more corrution might have creat in after the last check

Reply:
Done the "try by default"

Reply:
All done now